Makefile

```diff
@@ -13,6 +13,7 @@ GOOS = $(shell go env GOOS)
 # Deployment configuration
 CONTROLLER_NAMESPACE ?= open-cluster-management-agent-addon
 WATCH_NAMESPACE ?= managed
+MANAGED_CLUSTER_NAME ?= cluster1
```

Comment:
So that we don't have two variables to manage, I'm thinking this should work here:
```
MANAGED_CLUSTER_NAME ?= managed
WATCH_NAMESPACE ?= $(MANAGED_CLUSTER_NAME)
```

Reply:
done as suggested